packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -97,6 +112,7 @@ const getGroupedOptions = (): GroupedOptionArgs[] => {
 
 const optionSets = {
     [ExampleOptionsType.simpleOptions]: simpleOptions,
+    [ExampleOptionsType.placeholderOptions]: placeholderOptions,
```

Comment:
Did you intend for this to show up as a selectable option in the args table? If so can you give it a Sentence case name?

![image](https://github.com/user-attachments/assets/e5b45b89-72e6-46a6-acc3-5d06bc5e6e8e)

Reply:
Yes - done - 👍